### PR TITLE
Create dijkstra-algo.md

### PR DESCRIPTION
**Pull request by RohithaAiswarya16**
This PR adds an implementation of Dijkstra's Algorithm in C, C++, Python, and Java

### Description
This PR adds a README file for Dijkstra's Algorithm, providing a brief explanation of the algorithm, its time and space complexity, and applications. This documentation aims to help users understand the implementation and use cases of Dijkstra's Algorithm.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/graphs/dijkstra-algo.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+---
+id: dijkstra-algo
+title: Dijkstra's Algorithm
```
Comment (ajay-dhangar) [inline docs/graphs/dijkstra-algo.md]: replace `title: Dijkstra's Algorithm` with `title: "Dijkstra's Algorithm"`

File: docs/graphs/dijkstra-algo.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+sidebar_label: Dijkstra's Algorithm
```
Comment (ajay-dhangar) [inline docs/graphs/dijkstra-algo.md]: replace `sidebar_label: Dijkstra's Algorithm` with `sidebar_label: "Dijkstra's Algorithm"`

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by RohithaAiswarya16** [comment]
Thank you for reviewing, made the suggested changes, please let me know if there's more to do.
Thank you.

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**